mhvtl-kmod.spec.in

```diff
@@ -5,7 +5,7 @@
 %global kmodtool /usr/lib/rpm/redhat/kmodtool
 
 # If kversion isn't defined on the rpmbuild line, define it here.
-%{!?kversion: %define kversion 2.6.32-573.12.1.el6.%{_target_cpu}}
+%{!?kversion: %define kversion %(rpm -q kernel |sed -e 's/^kernel-//' |sed -e 's/.x86_64$//')}
```

Comment:
From scoutfs-kmod/utils 'indocker.sh':

```
rpm -q kernel --provides | awk '/kernel-uname-r/ {print $3}'
```

```
# rpm -q kernel --provides | awk '/kernel-uname-r/ {print $3}'
3.10.0-693.11.1.el7.x86_64
```

Just a slight tweak to getting the right kversion -- other RPM world things depend on this kernel-uname-r, so should remain correct in the future too.